conf/layer.conf

```
@@ -17,4 +17,4 @@ BBFILE_COLLECTIONS += "efibootguard"
 BBFILE_PATTERN_efibootguard = "^${LAYERDIR}/"
 BBFILE_PRIORITY_efibootguard = "6"
 
-LAYERSERIES_COMPAT_efibootguard = "hardknott honister kirkstone"
+LAYERSERIES_COMPAT_efibootguard = "scarthgap kirkstone"
```

usual sorting order is oldest first
```suggestion
LAYERSERIES_COMPAT_efibootguard = "kirkstone scarthgap"
```

---

ok